dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1881,7 +1881,8 @@ FROM
     ('trx-tron', 'ethereum', 'TRX', 0x50327c6c5a14dcade707abad2e27eb517df87ab5, 6),
     ('usdc-usd-coin', 'ethereum', 'aEthUSDC', 0x98c23e9d8f34fefb1b7bd6a91b7ff122f4e16f5c, 6),
     ('move-movement', 'ethereum', 'MOVE', 0x3073f7aaa4db83f95e9fff17424f71d4751a3073, 8),
-    ('sei-sei', 'ethereum', 'SEI', 0xbdf43ecadc5cef51b7d1772f722e40596bc1788b, 18)
+    ('sei-sei', 'ethereum', 'SEI', 0xbdf43ecadc5cef51b7d1772f722e40596bc1788b, 18),
+    ('usdm-mountain-protocol-usd', 'ethereum', 'USDM', 0x57f5e098cad7a3d1eed53991d4d66c45c9af7812, 18)
```

Comment:
```suggestion
    ('usdm-mountain-protocol-usd', 'ethereum', 'wUSDM', 0x57f5e098cad7a3d1eed53991d4d66c45c9af7812, 18)
```

Reply:
this address appears to be for wrapped version?

Reply:
sorry  that was a mistake from my end trying to copy the USDM address, fixed now ser!